static/js/src/intlTelInput.js

```diff
@@ -0,0 +1,20 @@
+import intlTelInput from "intl-tel-input";
+
+// Setup dial code dropdown options (intlTelInput)
+function setupIntlTelInput(phoneInput) {
+  const utilsScript =
+  "https://cdnjs.cloudflare.com/ajax/libs/intl-tel-input/17.0.18/js/utils.js";
```

Comment:
This makes the script dependant on an external source, when the package already includes this same script. The script isn't exposed an export for some reason, but we could include copying it to the project it as part of our `build-js` step, like we do for cookie-policy, global-nav and latest-news, see: https://github.com/canonical/ubuntu.com/blob/main/package.json#L13-L15

You should then be able to update this URL to the file within the project